roles/openshift_metrics/tasks/main.yaml

```diff
@@ -1,20 +1,24 @@
 ---
-# TODO import templates to this repo?
-- name: temporary hack to include templates from origin-metrics
-  fail: msg='the origin_metrics_dir variable is required'
-  when: "{{ origin_metrics_dir is not defined }}"
 - name: check that hawkular_metrics_hostname is set
   fail: msg='the hawkular_metrics_hostname variable is required'
   when: "{{ hawkular_metrics_hostname is not defined }}"
+- name: check the value of hawkular_cassandra_storage_type
+  fail:
+    msg: >
+      hawkular_cassandra_storage_type ({{ hawkular_cassandra_storage_type }})
+      is invalid, must be one of: emptydir, pv, dynamic
+  when: hawkular_cassandra_storage_type not in hawkular_cassandra_storage_types
 - name: Install Metrics
   include: "{{ role_path }}/tasks/install_{{ include_file }}.yaml"
-  vars:
-    template_dir: ../openshift_logging/templates
-    out_dir: /home/vagrant
   with_items:
     - metrics
     - heapster
     - hawkular
   loop_control:
     loop_var: include_file
-# TODO configure metricsPublicURL and restart master?
+- name: create objects
+  command: >
+    {{ openshift.common.client_binary }} -n '{{ metrics_project }}'
+    create -f {{ item }}
```

Comment:
i would recommend apply instead, create can fail if an object of the same name already exists; apply will try to resolve collisions and update if necessary